.drone/scripts/build-native.sh

```diff
@@ -1,7 +1,26 @@
 #!/usr/bin/env bash
 set -e
 
+# Get current package version
+version="$(cat .data.json | jq -r '.current_pkgver + "-" + .current_pkgrel')"
+
+# Update debian version
+DEBVERSION="$(cat debian/changelog | cut -f2 -d" " - | grep "(" | cut -f2 -d"(" | cut -f1 -d")")"
+
+# Setup env vars needed
+DEBFULLNAME="Leo Puvilland"
+DEBEMAIL="leo@craftcat.dev"
+
+rm debian/changelog
+
+if [ $version != $DEBVERSION ]; then
+  dch --create --package makedeb -D unstable -v $version "Initial release (Closes: #998039)."
+fi
```

Comment:
This shouldn't be needed since we use Git hooks to automatically set the version in `debian/changelog`. There could be cases where the user hasn't set up the Git hooks, but I don't want any extra bloat in the build scripts just to support that.